infrastructure/resource_group_level/modules/nat-gw_nat_ip.bicep

```diff
@@ -0,0 +1,56 @@
+param location string
+param appServicePlanName string
+
+var natGatewayName = 'tfs-${appServicePlanName}-nat-gw'
+var natGatewayIpAddressesName = 'tfs-${appServicePlanName}-nat-ip'
+
+// TODO:DTFS-6422 It looks like this is duplicated by the natGatewayIpAddresses below. Remove if all is ok.
```

Comment:
It does seem to work, but I'll check a bit more carefully before wiping this.
(It seems the bicep decompile will wire up dependencies from both sides - this duplication needs to be fixed by removing one wiring.)